templates/pagination.php

```diff
@@ -21,6 +21,13 @@
 ?>
 <nav class="job-manager-pagination">
 	<?php
+		/**
+	 	 * Filter the pagination arguments
```

Comment:
Thanks for the comment Jer.

>  In some cases, you'll find helpful to give the docblock a title, and then add more details on a new line below.

Can you give me an example or a few examples of this? Do you think the `@see` comment mentioned by Jake should suffice here or we need to do more?

I'd make the following change, let me know what you think of it. Bear with me as I may make some mistakes but I'll learn with time :)

```
/**
 * Filter the paginated link for job manager catalog pages.
 *
 * The paginate_links function can be used to create paginated link list for any area.
 * e.g.: « Prev 1 … 3 4 5 6 7 … 9 Next »
 *
 * @see https://codex.wordpress.org/Function_Reference/paginate_links
 *
 * @since 1.4.0
 *
 * @param array $args The pagination arguments.
 */
```

> The point here is not just to add documentation, but to make that documentation helpful to anyone searching through the filters later on.

I agree :)

Reply:
"for job manager catalog pages." is the most important change you've made here imo; it already clarifies things a lot, so folks will know what it is done right off the bat. The `@see` line gives all the other info you need. The rest of the description is not that useful then.

One last comment I would make would be about "job manager"; you'll probably want to set up some rules as to how you're going to name the plugin areas as you document those filters. Do you want to capitalize things every time for example?